release/models/mpls/openconfig-mpls-static.yang

```diff
@@ -217,6 +225,14 @@ submodule openconfig-mpls-static {
         "Next hop IP address for the LSP";
     }
 
+    leaf next-hop-network-instance {
```

Comment:
This leaf is still placed in the wrong grouping.

Please move it from `static-lsp-common-config` to `static-lsp-egress-config`